lineman/app/components/thread_page/activity_card/motion_edited_item_controller.coffee

```diff
@@ -0,0 +1,11 @@
+angular.module('loomioApp').controller 'MotionEditedItemController', ($scope) ->
+  $scope.proposal = $scope.event.version().proposal()
+  version = $scope.event.version()
+  $scope.proposalName =
+    if version.attributeEdited('name')
+      version.changes.name[1]
```

Comment:
I've been asking myself that ('Why are hash transforms so hard?') a lot recently.

Yea.. that's a good idea.. might do that.